proto/vtctldata.proto

```diff
@@ -811,6 +813,60 @@ message InitShardPrimaryResponse {
   repeated logutil.Event events = 1;
 }
 
+message MoveTablesCreateRequest {
+  // The necessary info gets passed on to each primary tablet involved
+  // in the workflow via the CreateVRWorkflow tabletmanager RPC.
+  string workflow = 1;
+  string source_keyspace = 2;
+  string target_keyspace = 3;
+  repeated string cells = 4;
+  repeated string tablet_types = 5;
+  repeated string source_shards = 6;
+  bool all_tables = 7;
+  repeated string include_tables = 8;
+  repeated string exclude_tables = 9;
+  // ExternalCluster is the name of the mounted cluster which has the source keyspace/db for this workflow
+  // it is of the type <cluster_type.cluster_name>
+  string external_cluster = 10;
```

Comment:
should we make this structured? e.g.

```protobuf
message VReplicationExternalCluster {
    string type = 1;
    string name = 2;
}

...

message MoveTablesCreateRequest {
    // ...
    VReplicationExternalCluster external_cluster = 10;
}
```

Reply:
It's only a name used to look it up in the topo server AFAIK.

Reply:
i still am not convinced that means it _shouldn't_ be structured, though

Reply:
I don't know what type is there, but we could have a message that just has `string name` in it. I don't feel strongly about it since it's been a simple string for years and migrate is rarely used (especially now that we have partial MoveTables).

Reply:
what i meant here was for

```
// it is of the type <cluster_type.cluster_name>
  string external_cluster = 10;
```

then type is cluster_type and name is cluster_name

Reply:
For MoveTables/Migrate, it's literally just a string name/handle/alias (the comment I copied from elsewhere) that maps to the simple string used to name the topo mount for the external cluster. I'll rename it to `external_cluster_name` -- it's nothing more than a simple arbitrary string that matches the simple string used for the topo mount command.

Reply:
All we're doing in VReplication is passing through the clusterName used here: https://github.com/vitessio/vitess/blob/main/go/vt/topo/external_vitess_cluster.go

And it's really only used for the `Migrate` command, which is a variation of MoveTables, but will need a new dedicated cli command (where that value is actually set).